spec/factories/ext_management_system.rb

```diff
@@ -258,7 +259,9 @@
   factory :ems_azure,
           :aliases => ["manageiq/providers/azure/cloud_manager"],
           :class   => "ManageIQ::Providers::Azure::CloudManager",
-          :parent  => :ems_cloud
+          :parent  => :ems_cloud do
+    provider_region { "us-east-1" }
```

Comment:
@djberg96 or @bronaghs or @agrare Is this a representativevalue for this provider type?